project.clj

```diff
@@ -48,25 +50,40 @@
                                        "TMP_DIR=\\$(mktemp -d) && \\
                                        wget -nv https://downloads.metabase.com/\\$METABASE_VERSION/metabase.jar -O \\$TMP_DIR/metabase.jar && \\
                                        mkdir -p repo && \\
-                                       mvn deploy:deploy-file -Durl=file:repo -DgroupId=com.firebolt -DartifactId=metabase-core -Dversion=1.40 -Dpackaging=jar -Dfile=\\$TMP_DIR/metabase.jar"]]}
+                                       mvn deploy:deploy-file -Durl=file:repo -DgroupId=com.firebolt -DartifactId=metabase-core -Dversion=1.40 -Dpackaging=jar -Dfile=\\$TMP_DIR/metabase.jar"]]
+            "sign" ["shell" "gpg" "-b" "-a" "--yes" "-u" ~(System/getenv "SIGN_KEY_ID")]
+            ;; lein just doesn't know how to deploy uberjar, so we have to do everything manually
+            "deploy-uberjar" ["do"
+                              "uberjar,"
+                              "pom,"
+                              "sign" ~(str uberjar-file ",")
+                              "sign" "pom.xml,"
+                              ;; There is some internal magic that renames pom.xml.asc to xml.asc during deploy,
+                              ;; so we have to hack it
+                              "shell" "mv" "pom.xml.asc" "pom.pom.asc,"
+                              "deploy" "releases" "io.firebolt/firebolt.metabase-driver" ~(str version)
```

Comment:
Do we still need this part if we're not deploying to maven anymore?

Reply:
I guess we can cleaup this stuff